modules/ui/raw_tag_editor.js

```diff
@@ -207,7 +207,8 @@ export function uiRawTagEditor(context) {
 
 
         function bindTypeahead(key, value) {
-            if (isReadOnly({ key: key })) return;
+            if (isReadOnly(key.datum())) return;
```

Comment:
are there any tests we can add for future modifications?

Reply:
Unfortunately I'm not familiar enough with JS nor with D3 so I won't be helping here. If and when you guys add some testing I will be glad to assist adding tests for that specifically.